content/source/docs/cloud/api/variable-sets.html.md

```diff
@@ -0,0 +1,230 @@
+---
+layout: "cloud"
+page_title: "Variable Sets - API Docs - Terraform Cloud and Terraform Enterprise"
+---
+
+# Variable Sets API
+
+This set of APIs covers create, update, list, delete, and some specialized operations.
+
+Viewing Variable Sets requires access to the owning organization. Destructive actions require org admin status or workspace management permissions.
+
+
+## Create a Variable Set
+`POST organizations/:organization_name/varsets`
+
+Example Payload:
+```json
+{:data=>
```

Comment:
This appears to be Ruby, rather than JSON.

Reply:
d'oh